src/handler.ts

```diff
@@ -46,12 +65,18 @@ export async function handlePullRequest (context: Context): Promise<void> {
     }
   }
 
-  if (config.addAssignees && reviewers.length > 0) {
+  if (config.addAssignees && (reviewers.length > 0 || Object.keys(config.assigneeGroups).length > 0)) {
```

Comment:
`if (config.addAssignees && (reviewers.length > 0 || Object.keys(config.reviewersGroups).length > 0)) {`

Reply:
what did you mean by this comment?

Reply:
I have changed the logic to be more straight forward